mautic/mautic
# Froala's froala_style.css removed - it was causing issues

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Related user documentation PR URL | /
| Related developer documentation PR URL | /
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/2445
| BC breaks? | Not known yet, but there might be different style of edited page/email now.
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
In the linked issue there was problem that Froala's CSS which was used to style the content while editing was causing wrong image sizes. It's not a Froala issue, but it was caused by the way Mautic handles assets. Froala's css file was renamed to less file so Mautic's grunt command would generate all less files it to one css file. Wile doing so, the CSS was changed from

`body img.fr-dii{margin: 5px auto; display: inline-block; float: none;}`
to
`body img.fr-dii{display: inline-block; float: none; vertical-align: bottom; margin-left: 5px; margin-right: 5px; max-width: calc(90%);}`

Which caused the image size issue.

This PR is more a hypotheses than a fix I'd be sure to apply, because it is against the [Froala's getting started guide](https://www.froala.com/wysiwyg-editor/docs) where the `froala_style` css should be loaded. So I don't know what can go wrong, but from my testing, it looks maybe better than when the style is loaded. Everything I tried is editable as before and it's cleaner experience. I'd appreciate as many tests as possible for this PR.

#### Steps to test this PR:
1. Apply this PR or copy-paste the full versions of the changed files to your Mautic's.
2. Repeat the steps. Now the image should have 600 px, the table borders are gone, but the table is still editable, because it highlights the `<td/>` you click on.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
Try to save this code as an email:
```
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head></head>
<body class="body" style="padding:0; margin:0; display:block; background:#ffffff; -webkit-text-size-adjust:none" bgcolor="#ffffff">
<table style="width: 600px;" class="full-width" align="center" border="0" cellpadding="0" cellspacing="0"> <tbody> <tr> <td class="image-100-percent" valign="top" style="font-family: 'Droid Sans', 'Helvetica Neue', 'Arial', 'sans-serif' !important;font-weight: 400;text-align: center;" data-slot-container> <div data-slot="image"> <img src="http://www.catster.com/wp-content/uploads/2015/06/bengal-kittens-05.jpg" width="600" height="600" alt="header image"> </div> </td> </tr> </tbody> </table>
</body>
</html>
```
You should see that the image is not full width of the table (600px), but less.
